.github/workflows/build.yml

```diff
@@ -66,3 +66,17 @@ jobs:
           BRANCH: asf-site
           FOLDER: dist
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Copy files from site to theme
+        run: ./site.sh prepare-theme
+      - name: Prepare theme package
+        working-directory: sphinx_airflow_thmee
+        run: python setup.py compile_assets sdist bdist_wheel
+      - uses: actions/upload-artifact@v2
+        name: Upload documentation as artifact
+        working-directory: sphinx_airflow_thmee
```

Comment:
The workflow is not valid. .github/workflows/build.yml (Line: 76, Col: 9): Unexpected value 'working-directory'